bolt/lib/Core/MCPlusBuilder.cpp

```diff
@@ -121,6 +130,11 @@ bool MCPlusBuilder::equals(const MCTargetExpr &A, const MCTargetExpr &B,
   llvm_unreachable("target-specific expressions are unsupported");
 }
 
+bool MCPlusBuilder::isTerminator(const MCInst &Inst) const {
+  return Analysis->isTerminator(Inst) ||
+         (opts::TerminalTrap && Info->get(Inst.getOpcode()).isTrap());
```

Comment:
@yota9 The new code makes `brk 0x3e8` Arm instruction a basic block terminator. If it's not the desired behavior, we can turn off `--terminal-trap` on Arm.

Reply:
@maksfb Hi! Currently I don't think it is a huge problem